RecoPixelVertexing/PixelVertexFinding/src/PixelVertexProducer.cc

```diff
@@ -41,7 +41,7 @@ PixelVertexProducer::PixelVertexProducer(const edm::ParameterSet& conf)
     track_pt_min   = PVcomparerPSet.getParameter<double>("track_pt_min");    
     if (track_pt_min != ptMin_) {
       if (track_pt_min < ptMin_)
-	edm::LogWarning("PixelVertexProducer") << "minimum track pT setting differs between PixelVertexProducer (" << ptMin_ << ") and PVcomparer (" << track_pt_min << ") [PVcomparer considers tracks w/ lower threshold than PixelVertexProducer does] !!!";
+	edm::LogInfo("PixelVertexProducer") << "minimum track pT setting differs between PixelVertexProducer (" << ptMin_ << ") and PVcomparer (" << track_pt_min << ") [PVcomparer considers tracks w/ lower threshold than PixelVertexProducer does] !!!";
```

Comment:
it seems we now agree the printout has accomplished its goal. if things return back to the state where it indicates a real unknown problem, it would then be good to reenable by default as a warning. Meanwhile we can more easily see other warnings

> On Oct 21, 2015, at 4:10 PM, arizzi <notifications@github.com> wrote:
> 
> In RecoPixelVertexing/PixelVertexFinding/src/PixelVertexProducer.cc:
> 
> > @@ -41,7 +41,7 @@ PixelVertexProducer::PixelVertexProducer(const edm::ParameterSet& conf)
> >      track_pt_min   = PVcomparerPSet.getParameter<double>("track_pt_min");    
> >      if (track_pt_min != ptMin_) {
> >        if (track_pt_min < ptMin_)
> > -	edm::LogWarning("PixelVertexProducer") << "minimum track pT setting differs between PixelVertexProducer (" << ptMin_ << ") and PVcomparer (" << track_pt_min << ") [PVcomparer considers tracks w/ lower threshold than PixelVertexProducer does] !!!";
> > +	edm::LogInfo("PixelVertexProducer") << "minimum track pT setting differs between PixelVertexProducer (" << ptMin_ << ") and PVcomparer (" << track_pt_min << ") [PVcomparer considers tracks w/ lower threshold than PixelVertexProducer does] !!!";
> 
> because nobody complained/noticed it earlier, and the fix is "fix the config".
> It is not "years" it is at most "1 year", in fact less given the the config change was integrated (checking JIRA) on Nov 5th 2014
> 
> —
> Reply to this email directly or view it on GitHub.
>